packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { useState, useEffect, useRef, forwardRef, useCallback, useImperativeHandle, useMemo } from 'react';
 
 // eslint-disable-next-line no-unused-vars
-import { EditorCommand, NoteBodyEditorProps } from '../../utils/types';
+import { EditorCommand, NoteBodyEditorProps, ResourceInfos } from '../../utils/types';
```

Comment:
Information is a non-countable noun. There is no plural of Info. May I suggest to change `ResourceInfos` to `ResourceInfo`

Reply:
I know that this is not really important, but it hurts me every time I see or read it.

Reply:
I have the same opinion with you. But, I think refactoring existing codes and changing name convention is out of the scope of this PR.

Reply:
Oh, sorry. I had the impression that this was introduced in this PR. Or at least some variables. My bad then.

Reply:
One `ResourceInfo`, multiple `ResourceInfos`. What's wrong with that :) Although I guess that's a bit a French thing because we say "les infos", which is the evening news.

I'd be curious though how would you name an array of "Info" objects.

Reply:
InfoMap or InfoArray. The thing is there is no plural of information in English. The word does not exist. But as I said, its not important.

other examples: aircraft, moose, fish, ice, species (although not all of these are non-countable)

e.g. Veracrypt uses the term dismount instead of unmount (volume). I explained to them that dismount is used for a horse or a bicycle, but not for a hard disk, volume or mount point. Either way, they still haven't fixed it and every time I read `dismount a volume` my entire body convulses.

Reply:
Yeah they are French too actually, always butchering the English language :) (We say "démonter") It's probably too late to rename "infos" but I'll consider that next time it comes up. Indeed perhaps the Map or Array suffix would make sense, or choosing a word that can have both a singular and plural form.